### ISSUE TITLE
Timer

### ISSUE THREAD

**Issue by GoogleCodeExporter** [AzoraFred/chronolapse#7]
```
A timer would be a nice feature to have, so that I could program
Chronolapse to start recording at a certain time and stop at another
automatically. This would be nice for Ludum Dare competitions for example.
```

Original issue reported on code.google.com by `pek...@gmail.com` on 28 Aug 2009 at 10:13

**Comment by GoogleCodeExporter** [comment]
Original comment by `kee...@gmail.com` on 1 Oct 2009 at 10:33
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Fixed/Added in version 1.0.3
```

Original comment by `kee...@gmail.com` on 23 Dec 2010 at 12:39
- Changed state: **Fixed**
- Added labels: **Milestone-Release-1.0.3**
